application/modules/statements/views/view.php

```diff
@@ -0,0 +1,227 @@
+<?php
```

Comment:
@pradeshc Please cleanup the file, remove commented statements and format the code correctly. There are a lot of wrong fields, labels that are from the quote view.

Reply:
Done